static/docs/user-guide/running-dvc-on-windows.md

```diff
@@ -0,0 +1,51 @@
+# Caveats running DVC on Windows
+
+There can be few performance issues while running DVC on Windows by default. For
+some of these issues there are workarounds or configuration options available.
+Below are some of these issues detected so far and their workarounds. If you
+find any, please let us know!
+
+DVC in general is slower on Windows as compared to other platforms (like Linux,
+MacOS, etc.) that show pretty good performance. Upon investigation, it turns out
+that most of the delays are coming from
+[NTFS](https://en.wikipedia.org/wiki/NTFS) (New Technology File System) not
+being very good at handling directories with large number of files. NTFS is a
+file system that the Windows NT operating system uses for storing and retrieving
+files on a hard disk.
+[Here](https://superuser.com/questions/15192/bad-ntfs-performance) is the
+resource for reference.
```

Comment:
this paragraph is a repetition of the first one and one of the below, I would suggest to remove it

Reply:
Done